readthedocs/doc_builder/python_environments.py

```diff
@@ -163,6 +163,63 @@ def install_core_requirements(self):
             '--no-cache-dir',
         ]
 
+        if self.project.has_feature(Feature.INSTALL_LATEST_CORE_REQUIREMENTS):
+            self._install_latest_requirements(pip_install_cmd)
+        else:
+            self._install_old_requirements(pip_install_cmd)
+
+    def _install_latest_requirements(self, pip_install_cmd):
+        """
+        Install all the latest core requirements.
+
+        By enabling the feature flag ``INSTALL_LATEST_CORE_REQUIREMENTS``
+        projects will automatically get installed all the latest core
+        requirements: pip, sphinx, mock, alabaster, setuptools, etc.
```

Comment:
In the docs above, it sounds like we aren't explicitly installing mock & alabaster. Should we clarify that here?